### PR TITLE
Update ios.markdown

### PR DESCRIPTION
**Pull request by towerhand**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by towerhand** [comment]
My first PR for a small change in the iOS docs, please let me know if I did something wrong,

File: source/_docs/ecosystem/ios.markdown [hunk 1]
```diff
@@ -31,7 +31,10 @@ The `ios` component is the companion component for the Home Assistant iOS app. W
 
 Loading the `ios` component will also load the [`device_tracker`](/components/device_tracker), [`zeroconf`](/components/zeroconf) and [`notify`](/components/notify) platforms.
 
-In order to save credentials and adjust settings within the iOS Home Assistant app, the optional [`api_password`](/components/http/#api_password) must be set. 
+Home Assistant for iOS app versions 1.5.0 and up supports the new authentication system introduced in Home Assistant 0.77.
```
Comment (arsaboo) [inline source/_docs/ecosystem/ios.markdown]: We can just say `Home Assistant app supports the new authentication system...`

File: source/_docs/ecosystem/ios.markdown [hunk 2]
```diff
@@ -31,7 +31,10 @@ The `ios` component is the companion component for the Home Assistant iOS app. W
...
+
+In order to save credentials and adjust settings within the Home Assistant for iOS app prior to version 1.5.0, the optional [`api_password`](/components/http/#api_password) must be set. (Not recommended)
```
Comment (arsaboo) [inline source/_docs/ecosystem/ios.markdown]: Here, we can just say "To use the legacy authentication... (not recommended)."

File: source/_docs/ecosystem/ios.markdown (hunk 1 above)
Comment (towerhand) [reply source/_docs/ecosystem/ios.markdown]: I think I got it.

File: source/_docs/ecosystem/ios.markdown [hunk 3]
```diff
@@ -31,7 +31,10 @@ The `ios` component is the companion component for the Home Assistant iOS app. W
 
 Loading the `ios` component will also load the [`device_tracker`](/components/device_tracker), [`zeroconf`](/components/zeroconf) and [`notify`](/components/notify) platforms.
 
-In order to save credentials and adjust settings within the iOS Home Assistant app, the optional [`api_password`](/components/http/#api_password) must be set. 
+Home Assistant for iOS app supports the new authentication system introduced in Home Assistant 0.77.
+
+In order to save credentials and adjust settings within the Home Assistant for iOS app prior to version 1.5.0, the optional [`api_password`](/components/http/#api_password) must be set. Using legacy authentication is not recommended.
```
Comment (arsaboo) [inline source/_docs/ecosystem/ios.markdown]: The app also supports `legacy API password` (https://www.home-assistant.io/docs/authentication/providers/#legacy-api-password), if you are still using it. Using legacy authentication is not recommended.

Comment (balloob) [reply source/_docs/ecosystem/ios.markdown]: Yeah we should not mention it in the docs in the intro, as people might think they need to include it.

Comment (towerhand) [reply source/_docs/ecosystem/ios.markdown]: Not sure how this works, can you guys make a commit to change things exactly how you want them? Or I gotta be the one making the changes?

Comment (frenck) [reply source/_docs/ecosystem/ios.markdown]: Removed the part as requested by balloob.

**Review by frenck — APPROVED** [review]

**Pull request merged**